backend/core/src/app.module.ts

```diff
@@ -41,12 +41,20 @@ import { CatchAllFilter } from "./shared/filters/catch-all-filter"
 export function applicationSetup(app: INestApplication) {
   const { httpAdapter } = app.get(HttpAdapterHost)
   const allowList = process.env.CORS_ORIGINS || []
+  const allowListRegex = process.env.CORS_REGEX !== "" ? JSON.parse(process.env.CORS_REGEX) : []
```

Comment:
The tests are failing because if `CORE_REGEX` doesn't exist in the env this check passes as true since `undefined !== ""`.

I think we could get away with just `const allowListRegex = process.env.CORS_REGEX ? JSON.parse(process.env.CORS_REGEX) : []`